docs/guide/algos.rst

```diff
@@ -15,6 +15,7 @@ DQN                  ❌           ✔️            ❌                ❌
 HER                  ✔️           ✔️            ❌                ❌               ❌
 PPO                  ✔️           ✔️            ✔️                 ✔️               ✔️
 QR-DQN [#f1]_        ❌          ️ ✔️            ❌                ❌               ✔️
+RecurrentPPO [#f1]_  ✔️           ✔️             ✔️                ✔️               ✔️
```

Comment:
NItpick, but DQN and PPO variants could be next to their respective source algorithms (DQNs next to each other, PPOs next to each other).

Reply:
the order is alphabetical, so it's easier to search for something normally ;)